### PR TITLE
Update form to include required fields

### PR DESCRIPTION
**Pull request by DavidKang**

### CODE REVIEW

**Comment by hennevogel** [comment]
Deploy preview for *obs-patterns* ready!

Built with commit ff4189eea8b1a67a7b03e476dbf2b7573fca4d10

https://deploy-preview-58--obs-patterns.netlify.com

File: _posts/molecules/2018-09-27-forms.html [hunk 1]
```diff
@@ -11,11 +11,13 @@
 %form
   .form-group
     %label{for: "exampleInputEmail1"} Email address
-    %input#exampleInputEmail1.form-control{"aria-describedby" => "emailHelp", placeholder: "Enter email", type: "email"}
+    %abbr.text-danger{ title: 'required' } *
+    %input#exampleInputEmail1.form-control{"aria-describedby" => "emailHelp", placeholder: "Enter email", type: "email", required: true}
```
Comment (dmarcoux) [inline _posts/molecules/2018-09-27-forms.html]: Space missing inside the brackets. Same for the line 20.

Comment (DavidKang) [reply _posts/molecules/2018-09-27-forms.html]: @dmarcoux, it was like that before, shall I add the space in all of them?

Comment (DavidKang) [reply _posts/molecules/2018-09-27-forms.html]: @dmarcoux, done ^^

**Review by dmarcoux — APPROVED** [review]

**Pull request merged**